### PR TITLE
Add large scale image search app WIP

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by jobergum** [comment]
Asking for a review of the sample code + schema. 

Please don't merge yet as we need a few more tests + @kkraune just re-organized testing SD pipelines.

**Comment by kkraune** [comment]
thanks, having a look now, doing the readme for some typos as well

File: billion-scale-image-search/src/main/application/schemas/image.sd [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+schema image {
+
+  document image {
+
+    # Don't need to guess language
+    field language type string {
+      indexing: "en" | set_language
+    }
```
Comment (kkraune) [inline billion-scale-image-search/src/main/application/schemas/image.sd:9]: https://docs.vespa.ai/en/linguistics.html

I think you can set this outside the document clause, but not sure if it makes a difference

File: billion-scale-image-search/README.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+</pre>
+
+## Download Vector Data
+
+These instruction uses the first split file (0000) of a total of 2314 files in the LAION2B-en split:
+
+Download the vector data file:
+
+<pre data-test="exec">
+$ wget \
```
[the remark this answers is not in the document]
Comment (jobergum) [reply billion-scale-image-search/README.md:180]: I had bad experiences with curl and this dataset, due to connectivity. Wget worked much better.

File: billion-scale-image-search/README.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+the `text` ranking profile:
+
+<pre data-test="exec">
+$ vespa query \
+ 'yql=select caption, url from image where nsfw contains "unlikely" and userQuery()'\
+ 'hits=10' \
+ 'query=two dogs running on a sandy beach' \
+ 'ranking=text'
+</pre>
+
```
[the remark this answers is not in the document]
Comment (jobergum) [reply billion-scale-image-search/README.md:304]: Currently we only have exported the text-encoder, passing raw image data to the container is not that easy. A notebook will demonstrate this, but it's 27MB large so I'm uncertain about bloating the sample app with it.

**Pull request merged**